server/repository/src/migrations/v2_09_00/add_shipped_number_of_packs_to_invoice_line.rs

```diff
@@ -30,29 +10,10 @@ impl MigrationFragment for Migrate {
         sql!(
             connection,
             r#"
-                ALTER TABLE invoice_line ADD COLUMN shipped_number_of_packs DOUBLE PRECISION;
+                ALTER TABLE invoice_line ADD COLUMN shipped_number_of_packs {DOUBLE};
```

Comment:
Nice, but what was the effect of 'DOUBLE PRECISION' before ?

Reply:
Seems to work ok in SQLite, but thought better to keep it consistent with current patterns.